tests/queries/0_stateless/02302_column_decl_null_before_defaul_value.sql

```diff
@@ -0,0 +1,51 @@
+select 'create table, column +type +NULL';
+DROP TABLE IF EXISTS null_before SYNC;
```

Comment:
Not really related but I didn't know about SYNC. It is undocumented at https://clickhouse.com/docs/en/sql-reference/statements/drop. Would be cool if you could add a brief note to the documentation too.

Reply:
You can create an issue for documentation